plugins/inputs/cloudwatch/README.md

```diff
@@ -101,6 +101,7 @@ API endpoint. In the following order the plugin will attempt to authenticate.
   #
   #  ## Dimension filters for Metric.  All dimensions defined for the metric names
   #  ## must be specified in order to retrieve the metric statistics.
+  #  ## 'value' has wildcard / 'glob' matching support.
```

Comment:
Could you please add a small example like
```suggestion
  #  ## 'value' has wildcard / 'glob' matching support such as `p-*`.
```
Furthermore, you should also add this to the `sampleConfig`...